<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StudentBasicInfo</class>
 <widget class="QWidget" name="StudentBasicInfo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1261</width>
    <height>797</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>23</y>
     <width>1051</width>
     <height>191</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="spacing">
     <number>5</number>
    </property>
    <property name="leftMargin">
     <number>10</number>
    </property>
    <property name="topMargin">
     <number>10</number>
    </property>
    <property name="rightMargin">
     <number>10</number>
    </property>
    <property name="bottomMargin">
     <number>10</number>
    </property>
    <item>
     <layout class="QGridLayout" name="gridLayout_5">
      <property name="sizeConstraint">
       <enum>QLayout::SetDefaultConstraint</enum>
      </property>
      <property name="leftMargin">
       <number>10</number>
      </property>
      <property name="topMargin">
       <number>10</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>10</number>
      </property>
      <property name="spacing">
       <number>10</number>
      </property>
      <item row="1" column="0" alignment="Qt::AlignRight">
       <widget class="QLabel" name="label_51">
        <property name="maximumSize">
         <size>
          <width>81</width>
          <height>41</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>幼圆</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>首个学期</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_50">
        <property name="maximumSize">
         <size>
          <width>81</width>
          <height>41</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>幼圆</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="2" column="0" alignment="Qt::AlignRight">
       <widget class="QLabel" name="label_49">
        <property name="maximumSize">
         <size>
          <width>63</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>幼圆</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>状态</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0" alignment="Qt::AlignRight">
       <widget class="QLabel" name="label_38">
        <property name="maximumSize">
         <size>
          <width>63</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>幼圆</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>分校</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="comboBox_campus">
        <property name="minimumSize">
         <size>
          <width>10</width>
          <height>31</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>121</width>
          <height>16777215</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="comboBox_first_semester">
        <property name="minimumSize">
         <size>
          <width>10</width>
          <height>31</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>121</width>
          <height>16777215</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QComboBox" name="comboBox_status">
        <property name="minimumSize">
         <size>
          <width>10</width>
          <height>31</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>121</width>
          <height>16777215</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QGridLayout" name="gridLayout_6">
      <property name="sizeConstraint">
       <enum>QLayout::SetDefaultConstraint</enum>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>10</number>
      </property>
      <property name="rightMargin">
       <number>10</number>
      </property>
      <property name="bottomMargin">
       <number>10</number>
      </property>
      <property name="spacing">
       <number>10</number>
      </property>
      <item row="1" column="0" alignment="Qt::AlignRight">
       <widget class="QLabel" name="label_48">
        <property name="maximumSize">
         <size>
          <width>63</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>幼圆</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>班主任</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0" alignment="Qt::AlignRight">
       <widget class="QLabel" name="label_52">
        <property name="maximumSize">
         <size>
          <width>63</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>幼圆</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>报名人</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0" alignment="Qt::AlignRight">
       <widget class="QLabel" name="label_53">
        <property name="maximumSize">
         <size>
          <width>81</width>
          <height>41</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>幼圆</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>层次</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0" alignment="Qt::AlignRight">
       <widget class="QLabel" name="label_54">
        <property name="maximumSize">
         <size>
          <width>81</width>
          <height>41</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>幼圆</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>学历形式</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="comboBox_education_type">
        <property name="minimumSize">
         <size>
          <width>10</width>
          <height>31</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>121</width>
          <height>16777215</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="comboBox_head_teacher">
        <property name="minimumSize">
         <size>
          <width>10</width>
          <height>31</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>121</width>
          <height>16777215</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QComboBox" name="comboBox_applicant">
        <property name="minimumSize">
         <size>
          <width>10</width>
          <height>31</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>121</width>
          <height>16777215</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QComboBox" name="comboBox_level">
        <property name="minimumSize">
         <size>
          <width>10</width>
          <height>31</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>121</width>
          <height>16777215</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QGridLayout" name="gridLayout_3">
      <property name="sizeConstraint">
       <enum>QLayout::SetDefaultConstraint</enum>
      </property>
      <property name="leftMargin">
       <number>10</number>
      </property>
      <property name="topMargin">
       <number>10</number>
      </property>
      <property name="rightMargin">
       <number>10</number>
      </property>
      <property name="bottomMargin">
       <number>10</number>
      </property>
      <property name="spacing">
       <number>10</number>
      </property>
      <item row="1" column="0" alignment="Qt::AlignRight">
       <widget class="QLabel" name="label_45">
        <property name="maximumSize">
         <size>
          <width>63</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>幼圆</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>企业</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0" alignment="Qt::AlignRight">
       <widget class="QLabel" name="label_46">
        <property name="maximumSize">
         <size>
          <width>63</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>幼圆</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>学期</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_47">
        <property name="maximumSize">
         <size>
          <width>81</width>
          <height>41</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>幼圆</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>推进进度</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0" alignment="Qt::AlignRight">
       <widget class="QLabel" name="label_37">
        <property name="maximumSize">
         <size>
          <width>63</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>幼圆</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>教学点</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="comboBox_teach_place">
        <property name="minimumSize">
         <size>
          <width>10</width>
          <height>31</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>121</width>
          <height>16777215</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="comboBox_campany">
        <property name="minimumSize">
         <size>
          <width>10</width>
          <height>31</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>121</width>
          <height>16777215</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QComboBox" name="comboBox_semester">
        <property name="minimumSize">
         <size>
          <width>10</width>
          <height>31</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>121</width>
          <height>16777215</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QComboBox" name="comboBox_advance_process">
        <property name="minimumSize">
         <size>
          <width>10</width>
          <height>31</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>121</width>
          <height>16777215</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QGridLayout" name="gridLayout_4">
      <property name="sizeConstraint">
       <enum>QLayout::SetDefaultConstraint</enum>
      </property>
      <property name="leftMargin">
       <number>10</number>
      </property>
      <property name="topMargin">
       <number>10</number>
      </property>
      <property name="rightMargin">
       <number>10</number>
      </property>
      <property name="bottomMargin">
       <number>10</number>
      </property>
      <property name="spacing">
       <number>10</number>
      </property>
      <item row="0" column="1">
       <widget class="QComboBox" name="comboBox_school">
        <property name="minimumSize">
         <size>
          <width>178</width>
          <height>31</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="comboBox_majoy">
        <property name="minimumSize">
         <size>
          <width>178</width>
          <height>31</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QComboBox" name="comboBox_kclass">
        <property name="minimumSize">
         <size>
          <width>178</width>
          <height>31</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="1" column="0" alignment="Qt::AlignRight">
       <widget class="QLabel" name="label_55">
        <property name="maximumSize">
         <size>
          <width>63</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>幼圆</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>专业</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0" alignment="Qt::AlignRight">
       <widget class="QLabel" name="label_56">
        <property name="maximumSize">
         <size>
          <width>63</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>幼圆</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>班级</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_57">
        <property name="maximumSize">
         <size>
          <width>81</width>
          <height>41</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>幼圆</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="0" column="0" alignment="Qt::AlignRight">
       <widget class="QLabel" name="label_39">
        <property name="maximumSize">
         <size>
          <width>63</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>幼圆</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>高校</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>240</y>
     <width>1051</width>
     <height>521</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>580</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>查询</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
